maths/armstrong_numbers.py

```diff
@@ -30,15 +30,19 @@ def armstrong_number(n):
         sum = sum + (rem ** number_of_digits)
         temp //= 10
     if n == sum:
-        print(n, "is an Armstrong number")
-    else:
-        print(n, "is not an Armstrong number")
+        return True
+
+    return False
 
 
 # In main function user inputs a number to find out if it's an Armstrong or not. Th function armstrong_number is called.
 def main():
     num = int(input("Enter a number to check if it is Armstrong or not: ").strip())
     armstrong_number(num)
+    if armstrong_number(num):

```

Comment:
We can lose line 41.

Reply:
`print(f"{num} is {'' if armstrong_number(num) else 'not '}an Armstrong number.")`